src/zope/pagetemplate/engine.py

```diff
@@ -320,9 +330,9 @@ class ZopeEngine(ZopeBaseEngine):
     wrapped in security proxies if the 'untrusted' extra is installed::
 
       >>> r = context.evaluate('python: {12: object()}.values')
-      >>> str(type(r).__name__) == (
-      ...   '_Proxy' if HAVE_UNTRUSTED else
-      ...   'builtin_function_or_method')
+      >>> str(type(r).__name__) in (
+      ...   ('_Proxy',) if HAVE_UNTRUSTED else
+      ...   ('builtin_function_or_method', 'method'))
       True
```

Comment:
WAAAAAAAAAA!  I need eye-bleach after looking at that ridiculous bit of nonsense.

Which is not to say that the change is wrong -- the original is a horrid example of why I hate doctests that try to exercise edge cases.